service/src/main/java/bio/terra/workspace/service/job/JobBuilder.java

```diff
@@ -141,10 +141,6 @@ private void populateInputParams() {
       throw new MissingRequiredFieldException("Missing flight class: flightClass");
     }
 
-    if (workspaceId == null) {
```

Comment:
this seems unrelated, is this supposed to be in this PR?

Reply:
no workspace id needs to be set for this flight.